autopush/web/base.py

```diff
@@ -163,6 +163,12 @@ def prepare(self):
                             ",".join(self.cors_request_headers))
             self.set_header("Access-Control-Expose-Headers",
                             ",".join(self.cors_response_headers))
+        if self.conf.sts_max_age:
+            self.set_header("Strict-Transport-Security",
+                            ";".join([
+                                "max-age={}".format(self.conf.sts_max_age),
```

Comment:
eh could have just done it all in the .format string but no biggie

Reply:
Thought about it. Decided on this partly because there's a few other potential options and the wrapping would have gotten ugly.